src/messages/index.ts

```diff
@@ -21,4 +23,9 @@ export const messages = {
   enabled: 'enabled',
   disabled: 'disabled',
   restart: '\'Restart\' for changes to take effect.',
+  ng2Detected: 'VSCode-Icons has detected an Angular 2 project. ' +
```

Comment:
I would stick to `Angular` without the version number as it will eventually be changing every 6 moths.